src/Resources/views/BlockAdmin/compose_preview.html.twig

```diff
@@ -8,13 +8,18 @@
        href="{{ admin_pool.getAdminByAdminCode('sonata.dashboard.admin.block').generateUrl('edit', { 'id': child.id, 'composer': true }) }}"
     >
         {% set service = attribute(blockServices, child.type) %}
+        {% if service.metadata is defined %}
+            {% set metadata = service.metadata %}
+        {% else %}
+            {% set metadata = service.blockMetadata %}
+        {% endif %}
```

Comment:
`{% set metadata = service.metadata|default(service.blockMetadata) %}`?

Reply:
I did the same suggestion on previous PRs, but I don't remember which was the problem preventing this expression.

Reply:
This is not possible :(
https://github.com/sonata-project/SonataPageBundle/pull/1064